app/_src/mesh/sbom.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Software Bill of Materials
+toc: false
+---
+
+A software bill of materials (SBOM) is an inventory of all software components (proprietary and open source), open source licenses, and dependencies in a given product. A software bill of materials (SBOM) provides visibility into the software supply chain and any license compliance, security, and quality risks that may exist.
+
+Starting in {{site.mesh_product_name}} 2.7.4, we are generating SBOMs for {{site.mesh_product_name}} and docker container images.
+
+1. Download the security assets for the required version of {{site.mesh_product_name}}
```

Comment:
Test link for `Preview Security assets for SBOM`:  [0.0.0-preview.vb402ff182 Security Asssets](https://cloudsmith.io/~kong/repos/kong-mesh-binaries-preview/packages/?q=name%3Asecurity-assets+version%3A0.0.0-preview.vb402ff182)